test/e2e/tests/resize_e2e_test.go

```diff
@@ -290,12 +293,18 @@ func validateRead(publishDir, testFileName, testFileContents string, instance *r
 	Expect(strings.TrimSpace(string(readContents))).To(Equal(testFileContents))
 }
 
-func validateVolumeStats(publishDir, volID string, tc *remote.TestContext) {
+func validateVolumeStats(publishDir, volID string, instance *remote.InstanceInfo, tc *remote.TestContext) {
+	// generate a random file between 1 Mb to 100 Mb for usage verification
+	rand.Seed(time.Now().UnixNano())
+	fileSize := 1 + rand.Int63n(100)
+	err := testutils.GenerateRandomFile(instance, publishDir, fileSize)
+	Expect(err).To(BeNil(), "Failed to generate file")
+
 	available, capacity, used, inodesFree, inodes, inodesUsed, err := tc.Client.NodeGetVolumeStats(volID, publishDir)
 	Expect(err).To(BeNil(), "Failed to get node volume stats: %v", err)
 	Expect(equalWithinEpsilon(available, minVolumeSize, defaultEpsilon)).To(BeTrue())
 	Expect(equalWithinEpsilon(capacity, minVolumeSize, defaultEpsilon)).To(BeTrue())
-	Expect(equalWithinEpsilon(used, 0, defaultEpsilon)).To(BeTrue())
+	Expect(used).To(Equal(util.MbToBytes(fileSize)))
```

Comment:
have you ran this to verify that the `used` be exactly the same with `fileSize` to the byte? do we need to add a small epsilon here of like 0.5%?

Reply:
Yes, it's verified. As long as the file size is a multiple of 1Mb, `NodeGetVolumeStats` will return the exact number we want.